routes/city.js

```diff
@@ -542,4 +537,36 @@ router.post("/spec-ds", tokenAuthentication, async (req, res) => {
   }
 });
 
+router.get("/jobs", async (req, res) => {
+  const input = req.query;
+  async function getJobs(params) {
```

Comment:
Remember the main backend rule - never trust the user!
You need to add logical iterations where function will check that request not empty - like if (!input) { say something to the user, he forgot the query}

Reply:
also will be better to check input thru RegExp - there is should be the name of the city without special characters

Reply:
okey, I understand and fixed it